test/server/cards/03_TWI/units/AdmiralYularenAdvisingCaution.spec.ts

```diff
@@ -0,0 +1,27 @@
+describe('Compassionate Senator', function () {
+    integration(function (contextRef) {
+        describe('Compassionate Senator\'s ability', function () {
+            it('should heal 2 damage from a unit or a base', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['echo-base-defender', 'admiral-yularen#advising-caution'],
+                    },
+                    player2: {
+                        groundArena: ['battlefield-marine'],
+                        hasInitiative: true,
+                    }
+                });
+
+                const { context } = contextRef;
+
+                // attack echo base defender with battlefield marine
```

Comment:
We shouldn't need any attacks here. Just check the HP of Echo Base is 4 and that the HP of Battlefield Marine is 3